ynr/templates/volunteer.html

```diff
@@ -28,21 +28,25 @@ <h3>You can help crowdsource candidates</h3>
 necessary for transparency and security of the database - all edits are logged and recorded on each candidate profile. 
 Edits are reviewed by Democracy Club staff, and users will be banned if vandalism is suspected.</p>
 
-<p>Now you need to find a candidate. You can find an existing candidate profile using the search function. Please make
-sure you have checked to see whether a profile already exists before creating a new candidate.</p>
+<h3>Finding and editing candidates</h3>
+<p>You can find an existing candidate profile using the search function at the top of this page. 
+Candidate profiles can be edited to add text and links using the ‘edit candidate button’, or you can add a photo using the separate button. </p>
 
-<p>When on a candidate page, you can use the View [candidate's] page on WhoCanIVoteFor.co.uk link to see the public-facing
-profile. Edits to Democracy Club Candidates can take between a few minutes and a couple of hours to update on
-WhoCanIVoteFor, depending on current site traffic.</p>
+<p>Please check to see whether a profile already exists before creating a new candidate.</p>
+
+<p>With the exception of Wikipedia, all links must by owned and run by the candidate or their party. 
+Text for personal statements will ideally be taken from these same sources, although statements submitted to third parties (eg. newspapers) can be acceptable.</p>
```

Comment:
I feel like we need to explain this a bit more clearly.

I know what you mean here, but it is a bit ambiguous. Facebook is run by meta, not the candidate or party, but https://www.facebook.com/MoseleyLibDems is an acceptable link. Lets make it clear.

Also, are we talking about the "links and social media" section, or source for edits?